.github/workflows/.mlc_config.json

```diff
@@ -0,0 +1,64 @@
+{
+    "ignorePatterns": [
+        {
+            "pattern": "^http://127.0.0.1"
+        },
+        {
+            "pattern": "^http://localhost"
+        },
+        {
+            "pattern": "^http://proxyserver"
+        },
+        {
```

Comment:
I ignored all the external links to avoid having CI broken by external changes. Should we consider adding a cronjob verifying those links?

Reply:
cronjobs checks are prone to be overlooked. I think we should add checking external links in PRs, but not make that check a required one for PR to be merged. This way we will notice issue earlier and not block PRs on it.